### ISSUE TITLE
Fix ImportOrganizerTest

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-xtend#403]
Fix ImportOrganizerTest 

http://services.typefox.io/open-source/jenkins/job/xtext-xtend/job/master/lastCompletedBuild/testReport/org.eclipse.xtend.core.tests.imports/ImportOrganizerTest/

cc @miklossy @kthoms the changes you did breaks these tests and i think some of them show a issue with your fix

**Comment by cdietrich** [comment]
e.g.

```
class Resource {
	def test() {
		val x = org.eclipse.emf.ecore.resource.Resource.Factory.Registry::INSTANCE
	}
}
```

the inner classes resource and the declaring one are not the same

**Comment by kthoms** [comment]
My bad. I executed the test before and after patch, and did not have that failure. Tamas, could you have a look please?

**Comment by miklossy** [comment]
Ok, I will take a look...

**Comment by cdietrich** [comment]
fixed in 2.14
